cmd/statreceiver/filter.go

```diff
@@ -0,0 +1,107 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package main
+
+import (
+	"regexp"
+	"time"
+)
+
+// PacketFilter is used during Packet parsing to determine if the Packet should
+// continue to be parsed.
+type PacketFilter struct {
+	appRE, instRE *regexp.Regexp
+}
+
+// NewPacketFilter creates a PacketFilter. It takes an application regular
+// expression and an instance regular expression. If the regular expression
+// is matched, the packet will be parsed.
+func NewPacketFilter(applicationRE, instanceRE string) *PacketFilter {
```

Comment:
nit -- "regex" is the common short of "regular expression". Param/field names could be "appRegex" and "instRegex". Can be applied to everywhere we have "re".

Reply:
`applicationPattern`, `instancePattern`?